docs/user_interface/how_to_install_ui_locally.rst

```diff
@@ -0,0 +1,53 @@
+.. _how-to-install-ui-locally-page:
+
+=========================
+How To Install UI Locally
+=========================
+
+These installation instructions are targeted towards developers and those who wish to deploy the desktop application.
+
+
+**Prerequisites**
+
+- Miniconda
+- npm
+
+
+Installing the UI
+-----------------
+
+1. Fork the repo on GitHub (your copy of the main repo)
```

Comment:
I think you should clarify that you are referring to the pareto-ui repo, not the project-pareto repo. I suggest including a link to the former.